file_loader.py

```diff
@@ -0,0 +1,69 @@
+import os
+import csv
+from db_connector import User
+import xlrd
+import pandas as pd
+
+
+class FileLoader(object):
+    def __init__(self, filename, file_extension):
+        self.filename = filename
+        self.file_extension = file_extension
+
+    # Create based on file extension :
+    @staticmethod
+    def factory(path_to_file):
+        filename, file_extension = os.path.splitext(path_to_file)
+        if file_extension == ".csv":
```

Comment:
You can use python dynamic behavior to get the class name from string (similar to ruby reflection). You can avoid the switch statement.